src/WingetCreateCLI/Common.cs

```diff
@@ -28,6 +28,37 @@ public static class Common
         /// </summary>
         public static string LocalAppStatePath => AppStatePathLazy.Value;
 
+        /// <summary>
+        /// Initiates the clean up process for the given directory.
+        /// </summary>
+        /// <param name="cleanUpDirectory">Directory to clean up.</param>
+        public static void BeginCleanUp(string cleanUpDirectory)
```

Comment:
Since this is a common function, it should be written in a way that could be used in other scenarios (Even if at this time it only applies to the cleanup settings). 

I would pass in the cleanupDays as a parameter instead of getting that value directly from settings within this method. Also rename the method so that it explicitly tells us what it is doing like `DeleteFilesOlderThan` (not the best name but just an example)

Reply:
Honestly couldn't think of a better name as well 😄 I went with `CleanUpFilesOlderThan`, made the function suitable for calling in other scenarios as well